src/main/resources/init_db.sql

```diff
@@ -0,0 +1 @@
+CREATE DATABASE `hibernate-relations-hw`;
```

Comment:
```suggestion
CREATE DATABASE `hibernate-relations-hw`;

```
make sure each file ends with exactly one empty line
![image](https://github.com/mate-academy/hibernate-relations-hw/assets/49767049/51aa5422-0bb5-4edd-a2eb-ffb04ce3f0f8)